src/home/home.scss

```diff
@@ -127,7 +127,7 @@
       background-color: $BG02;
       height: 12em;
       align-content: center;
-      justify-content: space-around;
+      justify-content: space-between;
```

Comment:
This does not result in what the wireframe is asking for.

Reply:
<img width="1239" alt="Screenshot 2021-05-31 at 16 35 47" src="https://user-images.githubusercontent.com/19796652/120215970-4f522600-c22e-11eb-88f6-4f3bd163c681.png">

Here is the result of the change

Reply:
take a thorough look  at the home page wireframe.  if you still don't see the problem, set up a zoom for us and you can show me.